contrib/devtools/split-debug.sh.in

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-
+set -e -o pipefail
```

Comment:
DrahtBot fails with

```
../contrib/devtools/split-debug.sh: 2: set: Illegal option -o pipefail

Reply:
On Ubuntu the default shell is dash (aka Debian Almquist Shell), to which /bin/sh is symlink. When your shell script is run with #!/bin/sh, you are effectively trying to run it with the default shell. However, dash doesn't have the pipefail option, which is why you're getting the error.

https://askubuntu.com/a/886540

Reply:
Oh crap. I hate shellscript programming so much.

Reply:
Oh wait, is this only about `split-debug.sh.in`, not the main gitian scripts? I don't think the `pipefail` matters there at all as the script doesn't use pipes.